src/scss/_main.scss

```diff
@@ -979,15 +979,6 @@ details.build-toggle {
   background: var(--color-focus);
 }
 
-.line a {
```

Comment:
this is not used, and was also conflicting with the new <a> tag. it should be safe to remove.

Reply:
i believe it was used prior to changing the line numbers to `<button>`